AUTHORS

```diff
@@ -3,6 +3,7 @@ Please keep this list in alphabetical order.
 
 Lillian Angel <langel@redhat.com>
 Andrew Azores <aazores@redhat.com>
+Bharathi BMS <bmsbharathi@gmail.com>
```

Comment:
While it is totally ok to add your name here the file is not really in sync anymore and I assume we will remove it in future :)